acsylla/__init__.py

```diff
@@ -1,7 +1,7 @@
 from . import errors
+from ._cython.cyacsylla import Cluster
```

Comment:
The idea is provide the interfaces of the objects like `basse.Cluster`, `base.Session`, and use the factories for getting an an instance that implements that interfaces, for example:

```python
from acsylla import (
    Cluster,
    Sesssion,
    create_cluster
)


cluster: Cluster = create_cluster(hosts)
session: Session = cluster.create_session()
```

So the general rule that I would like to keep is not providing directly any Cython object that can not provide documentation or typing to the user, so the the `acsylla.__init__` would need to follo that rule, that's the reasone of not providing the cython Cluster and asking the user to go through the `create_cluster` factory.

Is there any issue with the original version?

Reply:
Yup, that's fine. I can update it to work that way.

The benchmarks we failing trying to use the class instead of the factory.